File: jwt/src/main/java/com/member/jwt/dto/LoginRequestDto.java
package com.member.jwt.dto;

import jakarta.persistence.Entity;
import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import lombok.*;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class LoginRequestDto {

    @Email(message = "올바른 형식의 이메일 주소를 입력해주세요.")
    @NotBlank(message = "이메일을 입력해주세요.")
    private String email;

    @NotBlank(message = "비밀번호를 입력해주세요.")
    private String password;
}

File: jwtKotlin/src/main/kotlin/com/member/jwtkotlin/dto/LoginRequestDto.kt
package com.member.jwtkotlin.dto

import jakarta.validation.constraints.Email
import jakarta.validation.constraints.NotBlank

data class LoginRequestDto(
    @field:Email(message = "올바른 형식의 이메일 주소를 입력해주세요.")
    @field:NotBlank(message = "이메일을 입력해주세요.")
    val email: String? = null,

    @field:NotBlank(message = "비밀번호를 입력해주세요.")
    val password: String? = null
)
